src/app/@theme/components/nav-bar/nav-bar.component.scss

```diff
@@ -21,12 +33,32 @@
 .nav-itens {
     display: flex;
     justify-content: space-between;
-}
 
-.nav-itens li {
-    margin: 0 10px;
+    li {
+        margin: 0 10px;
+    }
+
+    li > a:not(.sair-btn) {
+        color: #7c8d93;
+
+        padding-bottom: 5px;
+
+        &:hover {
+            color: #006d77;
+            border-bottom: 3px solid #e29578;
+            font-weight: 500;
+            cursor: pointer;
+            transition: color 0.3s ease;
+        }
+    }
+
+    .sair-btn {
+        color: #e29578;
+    }
 }
 
-li {
-    list-style-type: none;
+.is-active {
+    color: #006d77 !important;
```

Comment:
oh no! a !important appears! É realmente necessário?

Reply:
Consegui resolver!!! Tive que adicionar a classe .is-active ao not()